trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapDates.js

```diff
@@ -16,15 +16,6 @@ const mapDates = (facility, facilitySnapshot, facilityTfm, dealDetails) => {
     facility,
   );
 
-  dates.isUsingFacilityEndDate = facilitySnapshot?.isUsingFacilityEndDate;
-  if (facilitySnapshot?.isUsingFacilityEndDate) {
-    dates.facilityEndDate = facilitySnapshot?.facilityEndDate;
-  }
-
-  if (facilitySnapshot?.isUsingFacilityEndDate === false) {
-    dates.bankReviewDate = facilitySnapshot?.bankReviewDate;
-  }
-
```

Comment:
This is no longer needed / should never have been in this file (the `mapDates` function is only called on non-GEF facilities)